prisma/schema.prisma

```diff
@@ -41,6 +41,8 @@ model Session {
 
 model User {
     id            String    @id @default(cuid())
+    publicKey     String
+    privateKey    String
```

Comment:
Storing an plaintext private key in a DB seems like a serious security vulnerability, no?

Reply:
It's a demo app, not a production one. Other alternative is to keep it in local storage.

Reply:
So I think we even can remove it because it can be accessed through the session object with jwt token.

Reply:
Fair enough! I appreciate the pushback 😄